packages/governance/src/contractHelper.js

```diff
@@ -49,6 +51,8 @@ const handleParamGovernance = (zcf, paramManager) => {
       ...originalPublicFacet,
       getSubscription: () => paramManager.getSubscription(),
       getContractGovernor: () => electionManager,
+      /** @type {GetGovernedVaultParams} */
+      // @ts-expect-error we know this ParamManagerFull is really a GetGovernedVaultParams
```

Comment:
Not for this PR, but...

I suspect there's a way to avoid type conflicts like this by parameterizing types such as `ParamManagerFull`.

It looks like I didn't file an issue, but I did some related work in https://github.com/Agoric/agoric-sdk/commits/claims-registar-dc-review esp 88b28e9 .